Library/Homebrew/dev-cmd/contributions.rb

```diff
@@ -0,0 +1,109 @@
+# typed: true
+# frozen_string_literal: true
+
+require "cli/parser"
+
+module Homebrew
+  extend T::Sig
+
+  module_function
+
+  SUPPORTED_REPOS = [
+    %w[brew core cask],
+    OFFICIAL_CMD_TAPS.keys.map { |t| t.delete_prefix("homebrew/") },
+    OFFICIAL_CASK_TAPS.reject { |t| t == "cask" },
+  ].flatten.freeze
+
+  sig { returns(CLI::Parser) }
+  def contributions_args
+    Homebrew::CLI::Parser.new do
+      usage_banner "`contributions` <email|name>"
+      description <<~EOS
+        Contributions to Homebrew repos for a user.
+
+        The first argument is a name (e.g. "BrewTestBot") or an email address (e.g. "brewtestbot@brew.sh").
+      EOS
+
+      comma_array "--repositories",
+                  description: "Specify a comma-separated (no spaces) list of repositories to search. " \
+                               "Supported repositories: #{SUPPORTED_REPOS.join(", ")}. " \
+                               "Omitting this flag, or specifying `--repositories=all`, will search all repositories."
+      flag "--from=",
+           description: "Date (ISO-8601 format) to start searching contributions."
+
+      flag "--to=",
+           description: "Date (ISO-8601 format) to stop searching contributions."
+
+      named_args min: 1, max: 1
```

Comment:
Sorry, forgot about this in my last review

```suggestion
      named_args number: 1
```